README.md

```diff
@@ -36,6 +36,9 @@ database driver with extended features.
 const sqlx = require('sqlx')
 const client = sqlx.createClient({
   connection_timeout: 1000, // destroy connection on timeout
+  logger: logFunction,      // if a log function is provided,
```

Comment:
other types of logging may be introduced in the future, and they seem have different parameters